kubernetes/gke-prow-build/prow/crds/boskos.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
ArgoCD is complaining about empty crds folder. I thought it might ignore an empty folder.
/hold cancel

Reply:
The prow CRD is too big to be applied without server-side apply. context: https://github.com/kubernetes/k8s.io/pull/7141/commits/4d3858b0ecf494c32956461dc69cbe5d48be9c5f